avocado/core/settings.py

```diff
@@ -107,6 +107,8 @@ def convert_value_type(value, value_type):
             value_type = float
         elif value_type == 'list':
             value_type = list
+        elif value_type == 'path':
+            value_type = os.path.expanduser
```

Comment:
Does that really help? IMHO, "other exceptions" is such a vague statement that it doesn't add any value.